autoload/codi.vim

```diff
@@ -436,6 +436,7 @@ function! s:codi_handle_data(data, msg)
   let out = s:preprocess(a:msg, i)
 
   for line in split(out, "\n")
+    call codi#log('s:codi_handle_data(): inside out for loop: line: '.line)
```

Comment:
Perhaps instead of manually writing the function name, it would be useful to use `expand('<sfile>')` to let the logging function handle recording the call stack. see `:h <sfile>`.